content/docs/setup/kubernetes/quick-start-gke-dm/index.md

```diff
@@ -175,7 +175,7 @@ $ kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=
 then
 
 {{< text plain >}}
-http://localhost:3000/dashboard/db/istio-dashboard
+http://localhost:3000/dashboard/db/istio-workload-dashboard
```

Comment:
should this be istio-mesh-dashboard?

Reply:
@douglas-reid ya - that would be a better generalized dashboard.

Reply:
@agmsb image attached for usage in this PR for @douglas-reid's suggestion:

<img width="1426" alt="screen shot 2018-09-25 at 8 47 34 pm" src="https://user-images.githubusercontent.com/755849/46056212-6e569c80-c104-11e8-994b-d1a354877c7a.png">

Reply:
This image LGTM

Reply:
Updated to istio-mesh-dashboard